src/api/index.ts

```diff
@@ -22,5 +23,7 @@ export const api: Api = {
     value != null
       ? store.set(key, value).then(() => store.save())
       : store.delete(key).then(() => store.save()),
-  CXUrl: 'https://expert.ethz.ch',
+  //TODO how to switch to production during build??
```

Comment:
This is usually done through `process.env.NODE_ENV`, which e.g. Webpack then replaces at build time with a boolean. But apparently Vite has some own env variables: https://vitejs.dev/guide/env-and-mode.html

Reply:
Moved into own issue